### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+You can install PyMilvus ORM via `pip` or `pip3` for Python 3.6+:
+
+```shell
+$ pip3 install pymilvus-orm
+```
+
+You can install a specific version of PyMilvus ORM by:
+
+```shell
+$ pip install pymilvus-orm==0.0.1
```
Comment (XuanYang-cn) [inline README.md:37]: Maybe `pip3`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+```
+
+You can upgrade PyMilvus ORM to the latest version by:
+
+```shell
+$ pip install --upgrade pymilvus-orm
```
Comment (XuanYang-cn) [inline README.md:43]: Maybe pip3

File: README.md [hunk 3]
```diff
@@ -1,2 +1,77 @@
...
+
+The commits on the development branch of each version will be packaged and uploaded to Test PyPI. Development branches refer to branches such as 1.0 and 1.1, and version releases are generated from the development branches, such as 1.0.1 and 1.0.2.
+
+The package name generated by the development branch is x.y.z.dev<dist>, where <dist> is the number of commits that differ from the most recent release.
+
+For example, after the release of 1.0.1, two commits were submitted on the 1.0 branch. At this time, the automatic packaging version number of the development branch is 1.0.1.dev2.
+
+To install the package on test.pypi.org, you need to append the parameter --index-url after pip, for example:
+```shell
+$ python3 -m pip install --index-url https://test.pypi.org/simple/ pymilvus-orm
```
Comment (XuanYang-cn) [inline README.md:72]: Do we support this? @scsven

File: pytest.ini [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[pytest]
+log_format = [%(asctime)s-%(levelname)s-%(name)s]: %(message)s (%(filename)s:%(lineno)s)
+
+# cli arguments. `-x`-stop test when error occurred;
+addopts = -x
+
+testpaths = tests
+
+log_cli = true
+log_level = 10
```
Comment (XuanYang-cn) [inline pytest.ini:10]: We already have a pytest.ini file in tests/

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (scsven) [reply README.md:72]: Not support.

Comment (czs007) [reply README.md:72]: Auto packaging is not supported currently,  but will be supported in the future

**Review by XuanYang-cn — APPROVED** [review]

**Pull request merged**